katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -0,0 +1,1788 @@
+
+%global scl_ruby_bin /usr/bin/%{?scl:%{scl_prefix}}ruby
+%global scl_rake /usr/bin/%{?scl:%{scl_prefix}}rake
+
+%global gem_instdir %{gem_dir}/gems/%{gem_name}-%{mainver}%{?prever}
+%global gem_docdir %{gem_dir}/doc/%{gem_name}-%{mainver}%{?prever}
+%global gem_cache %{gem_dir}/cache/%{gem_name}-%{mainver}%{?prever}.gem
+%global gem_spec %{gem_dir}/specifications/%{gem_name}-%{mainver}%{?prever}.gemspec
+
+%define katello_ostree %{?scl_prefix}rubygem-%{gem_name}_ostree
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Summary: Katello
+
+Version: %{mainver}
+Release: %{?prever:0.}%{release}%{?prever}%{?dist}
+Group:   Development/Ruby
+License: Distributable
+URL:     http://www.katello.org
+Source0: https://rubygems.org/downloads/%{gem_name}-%{version}%{?prever}.gem
+
+Requires: katello-selinux
+Requires: foreman >= 1.17.0
+Requires: foreman-postgresql
+Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
+Requires: %{?scl_prefix}rubygem(bastion) >= 6.0.0
+Requires: %{?scl_prefix}rubygem(bastion) < 7.0.0
+Requires: %{?scl_prefix}rubygem(oauth)
+Requires: %{?scl_prefix}rubygem(rest-client)
+Requires: %{?scl_prefix}rubygem(rabl)
+Requires: %{?scl_prefix}rubygem(foreman_docker) >= 0.2.0
+Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.0
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
+Requires: %{?scl_prefix}rubygem(gettext_i18n_rails)
+Requires: %{?scl_prefix}rubygem(apipie-rails) >= 0.1.1
+Requires: %{?scl_prefix}rubygem(runcible) >= 2.0.0
+Requires: %{?scl_prefix}rubygem(anemone)
+Requires: %{?scl_prefix}rubygem(deface) >= 1.0.0
+Requires: %{?scl_prefix}rubygem(deface) < 2.0.0
+Requires: %{?scl_prefix}rubygem(qpid_messaging)
+Requires: %{?scl_prefix_ruby}rubygem(json)
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+
+BuildRequires: foreman-assets
+BuildRequires: foreman-plugin >= 1.11.0
+BuildRequires: %{?scl_prefix}rubygem(foreman_docker) >= 0.2.0
+BuildRequires: %{?scl_prefix}rubygem(angular-rails-templates) >= 0.0.4
+BuildRequires: %{?scl_prefix}rubygem(bastion) >= 6.0.0
+BuildRequires: %{?scl_prefix}rubygem(bastion) < 7.0.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8.0
+BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
+BuildRequires: %{?scl_prefix}rubygem(gettext_i18n_rails)
+BuildRequires: %{?scl_prefix}rubygem(apipie-rails) >= 0.1.1
+BuildRequires: %{?scl_prefix}rubygem(runcible) >= 2.0.0
+BuildRequires: %{?scl_prefix}rubygem(anemone)
+BuildRequires: %{?scl_prefix}rubygem(deface) >= 1.0.0
+BuildRequires: %{?scl_prefix}rubygem(deface) < 2.0.0
+BuildRequires: %{?scl_prefix}rubygem(qpid_messaging)
+BuildRequires: %{?scl_prefix_ror}rubygem(rails)
+BuildRequires: %{?scl_prefix_ruby}rubygem(json)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+
+#old dependencies that are no longer needed, remove in katello 2.6
```

Comment:
Is this still relevant? In foreman-packaging these are typically added to the ```tfm``` package so would that be a better place?

Reply:
I think they are not relevant. However, I would not put them in `tfm` if they were as it should be up to the plugin that requires additional stack items to deprecate IMO.